pkg/action/action.go

```diff
@@ -76,8 +81,29 @@ func generateData(driver drivers.Driver, t string) interface{} {
 		return gofakeit.Date()
 	case drivers.Year:
 		return gofakeit.Number(1901, 2155)
+	case drivers.XML:
+		xml, err := gofakeit.XML(&gofakeit.XMLOptions{
+			Type:          "single",
+			RootElement:   "xml",
+			RecordElement: "record",
+			RowCount:      2,
+			Indent:        true,
+			Fields: []gofakeit.Field{
+				{Name: "first_name", Function: "firstname"},
+				{Name: "last_name", Function: "lastname"},
+				{Name: "password", Function: "password", Params: map[string][]string{"special": {"false"}}},
+			},
+		})
+		if err == nil {
```

Comment:
Please, can you invert the logic of this if statement to return with nil if we got error, otherwise the value of the xml? Just to keep the consistency of the code 🙂